license-finder.go

```diff
@@ -33,6 +33,7 @@ func copyLicenses(licenseDirs []string) error {
 				}
 			}
 		}
+		filenames = deduplicate(filenames)
```

Comment:
May as well use a map the whole time here instead of a slice?

Reply:
Probably, but this is already implemented :grimacing: